lib/elixir/lib/map_set.ex

```diff
@@ -367,19 +373,22 @@ defmodule MapSet do
 
   defimpl Collectable do
     def into(original) do
-      {original, fn
-        map_set, {:cont, x} -> MapSet.put(map_set, x)
-        map_set, :done -> map_set
-        _, :halt -> :ok
-      end}
+      {
+        original,
+        fn
```

Comment:
Can you please break it apart? Something like:

```elixir
fun = fn
  map_set, {:cont, x} -> MapSet.put(map_set, x)
  map_set, :done -> map_set
  _, :halt -> :ok
end
{original, fun}
```

and then run the formatter again. Thank you!